server/auth_function/lambda_function.py

```diff
@@ -151,13 +160,14 @@ def populate_user_if_necessary(db_connection, event) -> None:
         VALUES( {user_type_code}, {user_guid}, {cognito_user_id}, {user_name},
         CURRENT_USER, CURRENT_DATE, CURRENT_USER, CURRENT_DATE)
         ON CONFLICT (user_type_code, lower(user_name)) DO
-        UPDATE SET user_guid = {user_guid},  cognito_user_id = {cognito_user_id};"""
+        UPDATE SET user_guid = {user_guid},  cognito_user_id = {cognito_user_id}, business_guid = {business_guid};"""
```

Comment:
I think this is how it should work, I'll verify it after we deploy to dev, to check the log and the db

Reply:
Yes, please confirm that the business_guid remains null for IDIR or BCSC user.